src/SFA.DAS.EmploymentCheck.AcceptanceTests/Infrastructure/DbConnection.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Data;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmploymentCheck.AcceptanceTests.Infrastructure
+{
+    public abstract class DbConnection
```

Comment:
We could probably use the base class in Shared Packages (SFA.DAS.Sql.Client) rather than create a new one here.